example_test.py

```diff
@@ -18,6 +18,13 @@
 """
 Test the :py:class`pyfakefs.example` module to demonstrate the usage of the
 :py:class`pyfakefs.fake_filesystem_unittest.TestCase` base class.
+
+Fake filesystem functions like CreateFile(), CreateDirectory() or CreateLink()
```

Comment:
Please add back ticks, as in \`CreateFile()\` to the function names to get them rendered as code.